src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala

```diff
@@ -36,7 +36,8 @@ class SmartNumberArbiterSpec extends WarpJUnitSpec with CorePersistenceAware {
   def testSmartNumberBehavior(): Unit = {
     val arbiter: SmartNumberArbiter = new SmartNumberArbiter
     // create 1000 gaussian numbers with mean 50
-    val responseTimes: List[Double] = List.fill(1000)(50 + (Random.nextGaussian * 4))
+    val random: Random = new Random(seed = 123456)
```

Comment:
Why are we adding this?

Reply:
i thought it would be nice to have a deterministic random number generator